components/counter-bar/src/test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { createMatchers } from 'jest-emotion';
+import * as emotion from 'emotion';
+import Component from '.';
+import { LINK_COLOUR, WHITE } from 'govuk-colours';
+
+expect.extend(createMatchers(emotion));
+
+const testCounters = [
+  { id: 'abc', name: 'abc', score: 1, },
+  { id: 'def', name: 'def', score: 3, },
+  { id: 'ghi', me: 'ghi', },
```

Comment:
`me`?

Reply:
Ha! The funny thing is that I  assumed this was in the original, because it proved quite handy to have one element without a valid `name` property, in order to be able to test `empty` and therefore `opacity`, (line 39) but now I see it was a mistake of mine! I could code it differently though.

Reply:
:grin:

i guess we could have this done intentionally to test the `empty` stuff - but if so we should put a comment on this line to indicate that's what's going on.

one other minor thing - i'd have thought that the trailing comma here would be against linting rules - with the rules we have in the sec-env that'd be triggering `comma-dangle`

Reply:
(the rules are such that the dangling comma is required then objects are over multiple lines, but single line objects get no dangling comma)

Reply:
I'll have a look into making that test clearer - maybe it would be better to just _omit_ the name on that one - that should be clear enough?  Re. the `comma-dangle`, will check on that...